src/connectors/hierarchical-menu/connectHierarchicalMenu.ts

```diff
@@ -174,7 +181,8 @@ const connectHierarchicalMenu: HierarchicalMenuConnector = function connectHiera
       limit = 10,
       showMore = false,
       showMoreLimit = 20,
-      sortBy = ['name:asc'],
+      sortBy = DEFAULT_SORT,
+      facetOrdering = sortBy === DEFAULT_SORT,
       transformItems = (items => items) as TransformItems<HierarchicalMenuItem>,
```

Comment:
In the RFC it was discussed we might want to output a warning to raise awareness on the overlap, has that been implemented anywhere?

Reply:
In the conclusion of the warning we went with: since this is enabled by default, we can't give a warning if facet ordering would apply but there's also facet ordering (since sortBy was the only way to disable). At this point however an option "facetOrdering" has been added, so a warning is possible again.